packages/cloudformation-router/index.js

```diff
@@ -0,0 +1,46 @@
+const defaults = {
+  routes: [],
+  notFoundResponse: ({ requestType }) => {
+    return {
+      Status: 'FAILED',
+      Reason: `Route ${requestType} does not exist. @middy/cloudformation-router`
+    }
+  }
+}
+const cloudformationCustomResourceRouteHandler = (opts = {}) => {
+  if (Array.isArray(opts)) {
+    opts = { routes: opts }
+  }
+  const { routes, notFoundResponse } = { ...defaults, ...opts }
+
+  const routesStatic = {}
+  for (const route of routes) {
+    const { requestType, handler } = route
+
+    // Static
+    routesStatic[requestType] = handler
+  }
+
+  return (event, context, abort) => {
+    const { RequestType: requestType } = event
+    if (!requestType) {
+      return notFoundResponse({ requestType })
```

Comment:
I would for example expect that here we report to cloudformation that we have a failure, because if we don't do that cloudformation will simply wait x time until it timeouts whilst we already know that we won't be handling the request